opentelemetry-appender-log/src/lib.rs

```diff
@@ -151,6 +151,7 @@ where
             log_record.set_severity_number(severity_of_level(record.level()));
             log_record.set_severity_text(record.level().as_str());
             log_record.set_body(AnyValue::from(record.args().to_string()));
+            log_record.set_timestamp(SystemTime::now());
```

Comment:
As per the [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-timestamp), this field should come from source (in this case `log` library), and should be empty otherwise.